src/vs/workbench/parts/extensions/electron-browser/extensionsViewlet.ts

```diff
@@ -468,12 +510,17 @@ export class ExtensionsViewlet extends ViewContainerViewlet implements IExtensio
 		return this.panels.reduce((count, view) => (<ExtensionsListView>view).count() + count, 0);
 	}
 
-	private onEscape(): void {
-		this.search('');
+	private autoComplete(query: string, position: number): { fullText: string, additionalText: string, overwrite: number }[] {
+		if (query.lastIndexOf('@', position - 1) === -1 || query.lastIndexOf('@', position - 1) < query.lastIndexOf(' ', position - 1)) { return []; }
+
+		let wordStart = query.lastIndexOf('@', position - 1) + 1;
+		let alreadyTypedCount = position - wordStart - 1;
+
+		return Query.autocompletions().map(replacement => ({ fullText: replacement, additionalText: replacement, overwrite: alreadyTypedCount }));
 	}
 
 	private onEnter(): void {
-		(<ExtensionsListView>this.panels[0]).select();
+		(<ExtensionsListView>this.panels[0]).focus();
```

Comment:
On enter, I see that we reach this line of code. But visually I dont see any focus changing. So what does this do?

Reply:
Select also didn't do anything... this is dependent on #53616. I could bring those changes into this PR and close that one?

Reply:
as discussed, lets not try to do anything on Enter or Escape and focus on getting the tab to work